initializer/main.go

```diff
@@ -39,6 +39,12 @@ func main() {
 		"The name of a pre-built Gecko image, either on the local Docker engine or in Docker Hub",
 	)
 
+	chitSpammerImageNameArg := flag.String(
+		"chit-spammer-image-name",
+		"",
```

Comment:
What happens when this doesn't exist, which will be the case when someone is running the test who's not Ava? Seems like we should explicitly guard against this in the test suite, not running that test if the image doesn't exist